### PR TITLE
Add printer profiles for Vivedino Troodon 2.0

### PR DESCRIPTION
**Pull request by YGK3D**
Contributing two printer profiles for the Vivedino Troodon 2.0: one for RepRap firmware and one for Klipper firmware. Includes printer images, bed texture images, print presets and filament presets. Presets are based on the Voron 2.4 profiles.

### CODE REVIEW

File: resources/web/image/printer/Troodon 2.0 - Klipper_cover.png
Comment (SoftFever) [inline resources/web/image/printer/Troodon 2.0 - Klipper_cover.png:1]: The latest OrcaSlicer now recommends putting cover images inside the vender folder.
Could you move these cover images to Vivedino folder?

**Review by SoftFever — CHANGES_REQUESTED** [review]

**Comment by SoftFever** [comment]
@YGK3D 
Thank you for the PR.
Can you take a look at the comments?

File: resources/web/image/printer/Troodon 2.0 - Klipper_cover.png
Comment (YGK3D) [reply resources/web/image/printer/Troodon 2.0 - Klipper_cover.png:1]: Thanks. I have moved them now.

**Review by SoftFever — APPROVED** [review]
LGTM

**Pull request merged**